.github/workflows/platform-install-advanced.yml

```diff
@@ -8,6 +8,9 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: qaframework-bdd-tests
```

Comment:
@kdaud  i see you are checking out the working directory which was the same thing done on line 46 , do you mind giving me the reason why.

Reply:
It is smarter to set the working directory for all the commands to be executed than the current setting how the directory is checked out.